### ISSUE TITLE
NVDA should read whatever text on the screen using object navigation especially in media player classic.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#891]
**Reported by RamboUdin on 2010-09-07 00:43**
Hi,
NVDA can't read some text such as the statusbar. For example, when in a program called Media Player Clasic, NVDA doesn't read the remaining/elapse time for the track. So, If NVDA can read that, i'll be happy.

**Comment by leonardder** [comment]
Tried to reproduce this with a recent version: 1.7.11.24 (strange enough this number is lower), and it seems that nvda+end now reads the menu bar.

**Comment by derekriemer** [comment]
@leonardder why did you delete a comment here?

**Comment by jcsteh** [comment]
Hmm. That suggests the menu bar is at the bottom of the screen.

P4 because there haven't been any other requests for this and it will require an application specific fix.

**Comment by leonardder** [comment]
Clarification for my deletion of a comment: it was actually a spam comment from the former ticket system that hasn't got removed. I intended to do this just silently, but it seemed Github kept track of it publicly, which I wasn't aware of.

**Comment by Adriani90** [comment]
@leonardder is there any change in Mediaplayer classic 1.8.6 regarding this issue? For what ever reason I cannot install the mediaplayer classic on my PC. The installer does not run.

**Comment by Adriani90** [comment]
Can anyone reproduce this in Madia player classic 1.8.8? This is the last version which came out in October 2019.

**Comment by Adriani90** [comment]
We do not have updates on this issue since years and I cannot reproduce it in the last version of Mediaplayer classic. Closing as abandoned.
